packages/titanium-loading-indicator/src/titanium-full-page-loading-indicator.ts

```diff
@@ -66,7 +66,7 @@ export class TitaniumFullPageLoadingIndicatorElement extends LitElement {
   private _close() {
     window.clearTimeout(this._openDelayTimer);
     const totalTimeOpened = performance.now() - this._timeOpen;
-    const closeDelay = totalTimeOpened > this._minTimeOpen ? 0 : this._minTimeOpen - totalTimeOpened || 0;
+    const closeDelay = Math.max(this._minTimeOpen - totalTimeOpened, 0);
```

Comment:
Is there a functional difference here?

Reply:
Math.max returns the bigger of the two numbers. It's functionally identical, I just find the way it was before to be difficult to read.

Reply:
I'm saying, I don't actually care about this. I'm happy to revert it if you like the other way better.

Reply:
sounds good just checking I wasn't missing something, I like the simplicity in this change